docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -198,43 +198,192 @@ This table can then be queried in Trino::
+the following table:
+
+.. list-table:: Cassandra type to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Cassandra database type
+    - Trino type
+    - Notes
+  * - ``ASCII``
+    - ``VARCHAR``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    - Blobs have the limitation that they cannot be empty.
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DECIMAL``
+    - ``DOUBLE``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``INET``
+    - ``IPADDRESS``
+    -
+  * - ``INT``
+    - ``INTEGER``
+    -
+  * - ``LIST<?>``
+    - ``VARCHAR``
+    - Any collection (``LIST``, ``MAP``, or ``SET``) can be designated as
+      `FROZEN
+      <https://docs.datastax.com/en/dse/6.8/cql/cql/cql_using/refCollectionType.html>`_.
+  * - ``MAP<?, ?>``
+    - ``VARCHAR``
+    -
+  * - ``SET<?>``
+    - ``VARCHAR``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``TEXT``
+    - ``VARCHAR``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP(3) WITH TIME ZONE``
+    -
+  * - ``TIMEUUID``
+    - ``UUID``
+    -
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``TUPLE``
+    - ``ROW`` with anonymous fields
+    -
+  * - ``UUID``
+    - ``UUID``
+    -
+  * - ``UDT``
+    - ``ROW`` with field names
+    -
+  * - ``VARCHAR``
+    - ``VARCHAR``
+    -
+  * - ``VARINT``
+    - ``VARCHAR``
+    -
+
+No other types are supported.
+
+Trino type to Cassandra type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Trino types to the corresponding Cassandra types according to
+the following table:
+
+.. list-table:: Trino type to Cassandra type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Trino type
+    - Cassandra database type
+    - Notes
+  * - ``VARCHAR``
+    - ``ASCII``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``VARBINARY``
+    - ``BLOB``
+    - Blobs have the limitation that they cannot be empty.
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DOUBLE``
+    - ``DECIMAL``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``REAL``
+    - ``FLOAT``
+    -
+  * - ``IPADDRESS``
+    - ``INET``
+    -
+  * - ``INTEGER``
+    - ``INT``
+    -
+  * - ``VARCHAR``
+    - ``LIST<?>``
+    - Any collection (``LIST``, ``MAP``, or ``SET``) can be designated as
+      `FROZEN
+      <https://docs.datastax.com/en/dse/6.8/cql/cql/cql_using/refCollectionType.html>`_.
+  * - ``VARCHAR``
+    - ``MAP<?, ?>``
+    -
+  * - ``VARCHAR``
+    - ``SET<?>``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``VARCHAR``
+    - ``TEXT``
+    -
+  * - ``TIMESTAMP(3) WITH TIME ZONE``
+    - ``TIMESTAMP``
+    -
+  * - ``UUID``
+    - ``TIMEUUID``
+    -
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``ROW`` with anonymous fields
+    - ``TUPLE``
```

Comment:
Put next to the other `ROW` entry to avoid confusion

Reply:
Yes will redo the table since the first one is alpha order and the reverse table is not.